cmd/cli/install.go

```diff
@@ -218,6 +220,7 @@ func (i *installCmd) resolveValues() (map[string]interface{}, error) {
 		fmt.Sprintf("OpenServiceMesh.enableEgress=%t", i.enableEgress),
 		fmt.Sprintf("OpenServiceMesh.meshCIDRRanges=%s", strings.Join(i.meshCIDRRanges, " ")),
 		fmt.Sprintf("OpenServiceMesh.deployJaeger=%t", i.deployJaeger),
+		fmt.Sprintf("OpenServiceMesh.envoyLogLevel=%s", i.envoyLogLevel),
```

Comment:
We should add a cli validation for the allowed values to ensure bad values are caught early on.

Reply:
added validations